pkgs/stdenv/generic/setup.sh

```diff
@@ -1348,6 +1348,10 @@ showPhaseFooter() {
 
 
 genericBuild() {
+    # variable used by our gzip wrapper to add -n.
+    # gzip is in common-path.nix and is added to nix-shell but we only want to change it's behaviour in nix builds. do not move to a setupHook in gzip.
+    export GZIP_NO_TIMESTAMPS=1
+
```

Comment:
Why is this no longer in `gzip`'s setupHook?

Reply:
Hooks are run in nix-shell